app/controllers/admin/base_controller.rb

```diff
@@ -3,6 +3,7 @@
 class Admin::BaseController < ApplicationController
   include Pundit
 
+  before_action :set_paper_trail_whodunnit
```

Comment:
Should this be the first action? What happens someone tries to view the page without logging in?

Reply:
I've moved it to the end